man/generateRdFiles.Rd

```diff
@@ -0,0 +1,89 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/PythonPkgWrapperUtils.R
+\name{generateRdFiles}
+\alias{generateRdFiles}
+\title{Generate .Rd files for Python classes and functions}
+\usage{
+generateRdFiles(srcRootDir, pyPkg, module, modifyFunctions = NULL,
+  modifyClasses = NULL, functionPrefix = NULL, keepContent = FALSE,
+  templateDir = NULL)
+}
+\arguments{
+\item{srcRootDir}{The directory of the R package}
+
+\item{pyPkg}{The Python package name}
+
+\item{module}{The Python module}
+
+\item{modifyFunctions}{Optional function to modify the returned functions}
+
+\item{modifyClasses}{Optional function to modify the returned classes}
+
+\item{functionPrefix}{Optional text to add to the name of the functions}
+
+\item{keepContent}{Optional wheather the existing files at the target directory
+should be kept}
+
+\item{templateDir}{Optional path to a template directory}
```

Comment:
what happens if you pass in NULL? A: uses templates found in the /templates/ folder.